src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -113,6 +113,14 @@ describe('test data', () => {
   it(`has at least one cluster without a version`, () => {
     expect(filterData({ version: [''] })).to.have.length.gte(1);
   });
+  it('has at least three enabled clusters with the same impacted date', () => {
+    expect(
+      values.filter((v) => v.impacted === '2022-06-28T22:00:00.000Z')
+    ).to.have.length(3);
```

Comment:
This is 3 and just 3, but the test says "at least". And why we need 3?

Reply:
@ikerreyes, hah, to be honest, I don't remember my intention, maybe I wanted to test the order of clusters with identical timestamps. But I find it extra, removed.